sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/Main.scala

```diff
@@ -24,8 +24,13 @@ import software.amazon.awssdk.core.SdkBytes
 import software.amazon.awssdk.services.kinesis.KinesisAsyncClient
 import software.amazon.awssdk.services.kinesis.model.PutRecordRequest
 import software.amazon.kinesis.common.KinesisClientUtil
+import com.permutive.pubsub.producer.grpc.{GooglePubsubProducer, PubsubProducerConfig}
+import com.permutive.pubsub.producer.Model.{ProjectId, Topic}
+import com.permutive.pubsub.producer.encoder.MessageEncoder
```

Comment:
Now that we support both Kinesis and PubSub, we should have a `core` modules like we do and then have a `kinesis` module with just AWS deps and a `pubsub` with just GCP deps, like we do on the rest of our estate. So there will be one Docker image for Kinesis and one for PubSub.

Reply:
What's the benefit of separating it out and publishing multiple images?

Reply:
`import allOfTheInternet` problem solved

Reply:
OK yeah that all makes sense. 

I don't disagree, but realistically I dunno when I'll get to that task. So I guess this should be shelved until that can be done, or we make an issue for it in a future release. 

I don't really mind which call we make, just realistically the azure stuff (along with a bunch of other straggler stuff) is going to take prio over this PR.

Reply:
I'm OK to be relaxed about this -- after all this app is a developer's tool, not a full-blown production application.

FWIW the RDB loader (production application!) has a single docker image for AWS and GCP.

Reply:
I made [an issue for this](https://github.com/snowplow-incubator/snowplow-event-generator/issues/49), so we don't lose track of the idea. :)